src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -490,7 +536,6 @@ public String toString() {
       return "StreamEntryID";
     }
   };
-  
```

Comment:
Keep this unchanged in this PR.

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0

(Sorry it was due to formatting operation)